lib/backdrop.dart

```diff
@@ -631,6 +681,105 @@ class BackdropAppBar extends StatelessWidget implements PreferredSizeWidget {
+/// that should be shown while the front layer is "inactive" (the back layer is
+/// "showing").
+///
+/// Usage example:
+/// ```dart
+/// BackdropScaffold(
+///   appBar: ...,
+///   backLayer: ...,
+///   subHeader: BackdropSubHeader(
+///     title: Text("Sub Header"),
+///   ),
+///   frontLayer: ...,
+/// )
+/// ```
+class BackdropSubHeader extends StatelessWidget {
+  /// The primary content of the sub-header.
+  final Widget title;
+
+  /// The divider that should be shown at the bottom of the sub-header.
+  final Widget divider;
+
+  /// Flag indicating whether the leading widget for the sub-header should be
+  /// automatically determined by [BackdropSubHeader].
+  ///
+  /// If set to `true`, a leading `Icon(Icons.keyboard_arrow_up)` is added to
+  /// the sub-header.
+  ///
+  /// Defaults to `false`.
+  final bool automaticallyImplyLeading;
+
+  /// Flag indicating whether the trailing widget for the sub-header should be
+  /// automatically determined by [BackdropSubHeader].
+  ///
+  /// If set to `true`, a trailing `Icon(Icons.keyboard_arrow_up)` is added to
+  /// the sub-header.
+  ///
+  /// Defaults to `true`.
+  final bool automaticallyImplyTrailing;
+
+  /// Widget to be shown as leading element to the sub-header. If set, the value
+  /// of [automaticallyImplyLeading] is ignored.
+  final Widget leading;
+
+  /// Widget to be shown as trailing element to the sub-header. If set, the value
+  /// of [automaticallyImplyTrailing] is ignored.
+  final Widget trailing;
+
+  /// Creates a [BackdropSubHeader] instance.
+  ///
+  /// The [title] argument must not be `null`.
+  const BackdropSubHeader({
+    Key key,
+    @required this.title,
+    this.divider,
+    this.automaticallyImplyLeading = false,
+    this.automaticallyImplyTrailing = true,
+    this.leading,
+    this.trailing,
+  })  : assert(title != null),
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    Widget _buildAutomaticLeadingOrTrailing(BuildContext context) =>
+        FadeTransition(
+          opacity: Tween(begin: 1.0, end: 0.0)
+              .animate(Backdrop.of(context).controller),
+          child: Icon(Icons.keyboard_arrow_up),
+        );
+
+    return Column(
+      mainAxisSize: MainAxisSize.min,
+      children: <Widget>[
+        Row(
+          children: <Widget>[
+            leading ??
+                (automaticallyImplyLeading
+                    ? _buildAutomaticLeadingOrTrailing(context)
+                    : Container()),
+            Expanded(
+              child: Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 4.0),
+                child: title,
+              ),
+            ),
+            trailing ??
+                (automaticallyImplyTrailing
+                    ? _buildAutomaticLeadingOrTrailing(context)
+                    : Container()),
+          ],
+        ),
+        divider ?? Divider(height: 4.0, indent: 16.0),
```

Comment:
Done this change in https://github.com/fluttercommunity/backdrop/pull/29/commits/7d7929f4c628a73a8d75775fddb2e307448ca995.

Reply:
@WieFel Can you check now that if our default `BackdropSubHeader` setup matches the one in MDG.

I think the following needs to still change or reviewed:
- default textStyle - As of now it is set to `Theme.of(context).textTheme.subtitle1`. But I think it is not correct. @WieFel or @felixwortmann Can you figure out which textTheme is used in MDG?
- left indentation in `divider`
- proper padding

Reply:
>     * default textStyle - As of now it is set to `Theme.of(context).textTheme.subtitle1`. But I think it is not correct. @WieFel or @felixwortmann Can you figure out which textTheme is used in MDG?
Cannot find anything concrete in backdrop's MDG. To me it seems fine as is. It looks equal to the text theme in MDG. @felixwortmann could you find anything more?
>     * left indentation in `divider`
Indent of 16 is fine, as the text also has a left padding of 16. However I think we should also set a default `endIndent: 16.0`, because the divider should be symmetric:
![Screenshot_20200604_201641](https://user-images.githubusercontent.com/8345062/83799735-87b83600-a6a6-11ea-981f-9d726f58a13b.png)
>     * proper padding
See https://github.com/fluttercommunity/backdrop/pull/29#discussion_r435458391

Reply:
Looks good to me

Reply:
I thought it was subtitle1 because in MDG they use "Subtitle" text in examples, Indicating the textTheme to be used.

Reply:
Done necessary changes. Please review.